src/components/FormInput.tsx

```diff
@@ -339,12 +338,7 @@ const FormInput = (props: InputProps) => {
                       >
                         <ChipText>{chipData.option}</ChipText>
                         {isSubmitted && chipData.state === 'submittedSelected' && (
-                          <CheckIconForChip
-                            type={IconTypes.DONE}
-                            fillColor={IconColors.SECONDARY_2}
-                            width="20px"
-                            height="20px"
-                          />
+                          <CheckIconForChip type={'done'} color={'secondary'} size={20} />
```

Comment:
```
const CheckIconSize = 20;
```
Can we define this?

Reply:
I prefer not to significantly impact the message-related components here due to potential conflicts. Let's address this later